pennylane_lightning/core/src/lightning_gpu.toml

```diff
@@ -0,0 +1,125 @@
+schema = 1
+
+[device]
+name = "lightning.gpu"
+precision = ['float32', 'float64']
+
+[operations]
+observables = [
+        "PauliX",
+        "PauliY",
+        "PauliZ",
+        "Hadamard",
+        "SparseHamiltonian",
+        "Hamiltonian",
+        "Hermitian",
+        "Identity",
+        "Sum",
+        "Prod",
+        "SProd",
+]
+
+# These are gates that do not need decomposition.
+# Also gates which are natively supported by the device.
+[[operations.gates]]
+native = [
+        "Identity",
+        "PauliX",
+        "PauliY",
+        "PauliZ",
+        "Hadamard",
+        "S",
+        "T",
+        "PhaseShift",
+        "RX",
+        "RY",
+        "RZ",
+        "Rot",
+        "CNOT",
+        "CY",
+        "CZ",
+        "SWAP",
+        "CSWAP",
+        "Toffoli",
+        "IsingXX",
+        "IsingXY",
+        "IsingYY",
+        "IsingZZ",
+        "ControlledPhaseShift",
+        "CRX",
+        "CRY",
+        "CRZ",
+        "CRot",
+        "SingleExcitation",
+        "SingleExcitationPlus",
+        "SingleExcitationMinus",
+        "DoubleExcitation",
+        "DoubleExcitationPlus",
+        "DoubleExcitationMinus",
+        "MultiRZ",
+
+	# QubitUnitary is technically
+	# "applyMatrix", but it doesn't
+	# belong in matrix gates.
+        "QubitUnitary",
+
+	# All adjoint operations in lightning_kokkos
+	# are applied via the inverse flag.
+        "Adjoint(S)",
+        "Adjoint(T)",
+        "Adjoint(SX)",
+        "Adjoint(ISWAP)",
+        "Adjoint(SISWAP)",
```

Comment:
Why include these five adjoint here? Should we not list them all, or none since they can be applied with the optional argument `inverse` as in the comment?

Reply:
I moved them to native gates.

Reply:
@vincentmr Do you know why lightning lists these at all actually in its device class [here](https://github.com/PennyLaneAI/pennylane-lightning/blob/502fecbec6940cc710a209592cbdf178f8a02448/pennylane_lightning/lightning_qubit/lightning_qubit.py#L92)? Given that lightning can apply the adjoint of any gate it natively supports, including these five seems a bit arbitrary (maybe also redundant, at least I would hope it wouldn't need to list three copies for every gate: regular, adjoint, controlled).

Reply:
I don't know why. I feel the same, but these predates me so I don't know why these five are there in the first place. As for duplication, I don't know for adjoint, but PennyLane does not assume controlled versions are supported on the device. So if, say, `C(SingleExcitation)` is not listed PennyLane will try to decompose it. That's why all the controlled gates are listed.

Reply:
Interesting, should this list of five be extended to all the gates that have a native kernel then? We would need to make the corresponding change in `lightning_qubit.py` as well.

Reply:
Closing this comment. Feel free to reopen.